phobos/core/robot.py

```diff
@@ -20,6 +20,9 @@
 from ..utils.transform import create_transformation, inv, get_adjoint
 from ..utils.tree import find_close_ancestor_links
 from ..utils.urdf import read_urdf_filename, create_pdf_from_urdf, transform_object, get_joint_info_dict
+from ..utils.commandline_logging import get_logger, setup_logger_level
+
+log = setup_logger_level()
```

Comment:
If we do this in every file, aren't we generating lots of logger instances, that mess up the memory and are not in sync?
Or does setup_logger_level take care of this?

Reply:
Ok i found further down that the logger takes care of this. Anyways, why are we calling setup_logger_level here and not somewhere more basic? like in scripts/phobos.py e.g. And here we only use that logger...